src/mame/drivers/magicard.cpp

```diff
@@ -1331,26 +1952,27 @@ ROM_END
   |                                                                                                                |
-  |      __________                          _____________               ___________        ___                    |
-  |     |  74LS04  |                        |LC324256BP-70|             |     C     |      |   |     ___           |__
-  |     |__________|                        |_____________|             |___________|      |EMP|    |. .|           __|
+  |      __________                          _____________               ___________        ___                    |                            
+  |     |  74LS04  |                        |LC324256BP-70|             |     C     |      |   |     ___           |__ 
+  |     |__________|                        |_____________|             |___________|      |EMP|    |. .|           __|           
   |                                          _____________                                 |TY |    |. .|           __|
   |            ____                         |LC324256BP-70|                                |   |    |. .|           __|
   |           | A  |                        |_____________|                                |SOC|    |. .|           __|
   |           |____|                         _____________                                 |KET|    |. .|           __|
   |                                         |LC324256BP-70|                                |___|    |. .|           __|
   |                                         |_____________|           __________________            |. .|           __|
-  |                                          _____________           |   ADV476KN35E    |           |. .|           __|
+  |                                          _____________           |   ADV476KN35E    |           |. .|           __|    
   |                                         |LC324256BP-70|          |                  |           |___|           __|
   | _______                                 |_____________|          |     OF19802.3    |                          |
   ||DS1207 |                                                         |__________________|                          |
-  ||_______|                                                                                                       |
-  |         ___                       XTAL2                              __________                                |
-  |        |   |        ________          ________________              | PIC16C54 |                               |
+  ||_______|                                                                                                       |                       
+  |         ___                       XTAL2                              __________                                |   
+  |        |   |        ________          ________________              | PIC16C54 |                               |        
```

Comment:
There seem to be a lot of lines changed just by introducing trailing whitespace that will be removed, causing more churn.